docs/how-to/build-interacting-with-the-network.md

```diff
@@ -5,20 +5,29 @@ description: Learn how to interface with the Filecoin network.
 
 # Interacting with the Network
 
-**TODO: Replace content.** - Note that second-level headers are surfaced in the left nav. 
+This page outlines various options for connecting to networks while building and operating your service or application.
+
+Each Filecoin-based service or application will need to use at least one Filecoin node that maintains consensus. All interactions with the network must flow through an up-to-date node: sending and receiving market deals, sending and receiving data, and more.
+
+You can run this node yourself or choose a hosted option. Running your own node offers more decentralized control, but it requires non-trivial time and resources. Choosing a hosted node makes it easier to get started quickly, but you’ll need to trust the hosting provider.
 
 ## Local networks
```

Comment:
Just confirming you intended to leave this as a level 2 header and have it surfaced in the nav?

Reply:
Yes for here